gulpfile.ts

```diff
@@ -1,4 +1,6 @@
-import {Gulpclass, Task, SequenceTask, MergedTask} from "gulpclass";
+import { Gulpclass, Task, SequenceTask, MergedTask } from "gulpclass";
+
+declare function require(module: string): any;
```

Comment:
hmm it should work without problems without this line. Maybe you have something outdated (including globally installed ts-node)? Or maybe old version of typescript, or your ide is not set to use latest version of compiler? Because it does compile without any errors on my local and CI too. 

```typescript
$ gulp tests
[09:21:18] Requiring external module ts-node/register
[09:21:21] Using gulpfile ~/www/opensource/typeorm/typeorm/gulpfile.ts
[09:21:21] Starting 'tests'...
[09:21:21] Starting 'compile'...
[09:21:28] Finished 'compile' after 7.22 s
[09:21:28] Starting 'tslint'...
[09:21:32] Finished 'tslint' after 3.8 s
[09:21:32] Starting 'coveragePre'...
[09:21:33] Finished 'coveragePre' after 908 ms
[09:21:33] Starting 'coveragePost'...
...
[09:22:32] Finished 'coveragePost' after 59 s
[09:22:32] Starting 'coverageRemap'...
[09:22:32] Finished 'coverageRemap' after 674 ms
[09:22:32] Finished 'tests' after 1.18 min
```

Everything else in this PR looks good